spec/surfnperf_spec.js

```diff
@@ -1096,5 +1096,74 @@ define('spec/surfnperf_spec', [
 
     });
 
+
+    describe('#chromeLoadTimes', function() {
+      it('returns a reference to window.chrome.loadTimes() if the browser supports it, otherwise undefined', function() {
+        if(window.chrome && window.chrome.loadTimes()) {
+          expect(SurfNPerf.chromeLoadTimes()).toEqual(window.chrome.loadTimes());
+        } else {
+          expect(SurfNPerf.chromeLoadTimes()).toEqual(undefined);
+        }
+      });
+    });
+
+    describe('#msFirstPaint', function() {
+      it('returns a reference to window.performance.timing.msFirstPaint if the browser supports it, otherwise undefined', function() {
+        if(window.performance.timing.msFirstPaint) {
```

Comment:
here you have to check `if(window.performance && window.performance.timing && window.performance.timing.msFirstPaint) {`